connector/connector.py

```diff
@@ -443,10 +443,10 @@ def unwrap_binding(self, binding_id, browse=False):
         else:
             binding = self.model.browse(binding_id)
 
-        openerp_record = getattr(binding, self._openerp_field)
+        record = getattr(binding, self._openerp_field)
```

Comment:
@lmignon - Seems like a best practice to use item lookup instead of attribute for recordsets IMO, but you are correct that it's not editable in the UI. I'll remove the PR block